### PR TITLE
cleaning how to contribute

### PR DESCRIPTION
**Pull request by vinodkahuja**
Deleting unnecessary files from how to contribute section. Only three files will be kept in how to contribute section and those are.
1. Quickstart for Newcomers
2. Contributor Roadmap
3. DCO Setup

Signed-off-by: Vinod K. Ahuja <vahuja@unomaha.edu>

### CODE REVIEW

File: how-to-contribute/development-contribution.md [hunk 1]
```diff
@@ -1,125 +0,0 @@
-
```
Comment (sgoggins) [inline how-to-contribute/development-contribution.md:1]: I don't think we should take this out until we have something to replacement. There's a design decisions doc in development, as well as a comparison doc that Dawn, Daniel and Sean are working on.

**Comment by geekygirldawn** [comment]
I'm always a bit concerned about deleting files and directories, since we should always assume that other people will have linked to those pages from their blogs, websites, social media, and other posts.

It might be better to leave at least some of those pages intact in the repo, but replace the contents with a link to where the new content can be found?

**Comment by germonprez** [comment]
@vinodkahuja -- thoughts on this?

**Comment by vinodkahuja** [comment]
@sgoggins @geekygirldawn In the KB review meeting, the group decided to use KB as a pointer for newcomers on how to get started and where to contribute to different sections and not to provide technical details like what tools, programs, and frameworks to use to develop Augur, Grimoirelab or Cregit. That is why this file is proposed for deletion. A new PR # 539 is created where we have pointed to newcomers on where they can contribute and get more details. These technical details should reside either in Augur,  Grimoirelab, or Cregit repos rather than in KB.

Further, in the KB review meeting, each individual file in this PR was reviewed by participants in those meetings, and then it was decided which file to keep and which to delete. The exact detail of decision on each file to delete, relocate, and merge is available in [Google docs here](https://docs.google.com/spreadsheets/d/1dEMY-vtiGPdM4N4poyi_zy4jKdYloWycJl23YfTzCU8/edit?pli=1#gid=0). I have followed the same and deleted or moved the content that the group decided and that was the point of reviewing, cleaning, and simplifying the KB.

**Comment by geekygirldawn** [comment]
In general, it's not great practice to delete anything with a URL, especially documents that have had helpful information in the past. The reason is that other people may have linked to these pages from blog posts / websites or bookmarked these pages in the past. If you delete them, the user is left with nothing and no idea where that information might be found now. It's more user friendly to remove the contents of the file (leaving the file itself intact) and replace the old information with a link to where that user can go to get similar information in the future. In some of these cases, you might want to direct them to one of the software repos, software docs, or to another page in the knowledge base.

In general, I try to avoid ever deleting anything with a URL without some way of redirecting people to somewhere useful where they can find similar, but updated information, instead of hitting a dead end in a 404.

**Pull request closed without merging**